lib/ui/settings/change_password/password_success_screen.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
```

Comment:
We might not need a new success screen, this is because we already have one in the app hence it will be great to reuse it.